backend/src/main/java/com/bakdata/conquery/apiv1/query/concept/specific/external/CQExternal.java

```diff
@@ -78,42 +92,87 @@ public class CQExternal extends CQElement {
 	 * @implNote FK: I would prefer to implement this as a guava table, but they cannot be deserialized with Jackson so we implement the Table manually.
 	 */
 	@InternalOnly
-	@Getter
+	@Getter(AccessLevel.PRIVATE)
 	private Map<Integer, Map<String, List<String>>> extra;
 
-	public CQExternal(@NotEmpty List<String> format, @NotEmpty String[][] values) {
+	public CQExternal(List<String> format, @NotEmpty String[][] values, boolean onlySingles) {
 		this.format = format;
 		this.values = values;
+		this.onlySingles = onlySingles;
 	}
 
+	@JsonIgnore
+	@ValidationMethod(message = "Headers are not unique")
+	@SuppressWarnings("unused")
```

Comment:
Das habe ich IntelliJ beigebracht, dass ValidationMethod nicht unused ist. Finde ich persönlich besser als 3 Annos pro Validation Method.

Reply:
ah gut. In dem fall, hat die annotation auch gar nichts gemacht :D